### PR TITLE
*: fix yaml indent issues

### PR DESCRIPTION
**Pull request by CharLotteiu**
### What is changed, added, or deleted? (Required)

Fix yaml indent issues to close https://github.com/pingcap/docs-dm/issues/676.

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v3.0 changes)
- [x] v2.0 (TiDB DM 2.0 versions)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by TomShawn — APPROVED** [review]

**Comment by GMHDBJD** [comment]
intent:
  https://github.com/pingcap/docs-dm/blame/master/en/quick-start-create-task.md#L180
  https://github.com/pingcap/docs-dm/blame/master/en/quick-start-create-task.md#L186
  https://github.com/pingcap/docs-dm/blame/master/en/quick-start-create-task.md#L186
wrong config:
  https://github.com/pingcap/docs-dm/blame/73e4d096eb0c38a268758efc1e29fd274ccfaabe/en/task-configuration-file-full.md#L127 should be ["filter-rule-1", "filter-rule-2"]
 Chinese version should also be updated

File: en/quick-start-create-task.md [hunk 1]
```diff
@@ -158,37 +158,37 @@ Now, suppose that you need to migrate these sharded tables to the `db_target.t_t
...
         mydumper-thread: 4
         loader-thread: 16
         syncer-thread: 16
     block-allow-list:  # This configuration applies to DM versions higher than v2.0.0-beta.2. Use black-white-list otherwise.
-    instance:
+      instance:
         do-dbs: ["~^sharding[\\d]+"]
         do-tables:
         -  db-name: "~^sharding[\\d]+"
         tbl-name: "~^t[\\d]+"
```
Comment (GMHDBJD) [inline en/quick-start-create-task.md:184]:
```suggestion
        - db-name: "~^sharding[\\d]+"
          tbl-name: "~^t[\\d]+"
```

**Review by GMHDBJD — APPROVED** [review]
/lgtm

**Pull request merged**